packages/cli/src/extended-error.ts

```diff
@@ -0,0 +1,19 @@
+import { CheckupError } from '@checkup/core';
+import { red } from 'chalk';
+
+export function extendedError(
+  error: string | Error | CheckupError,
+  options: { code?: string; exit?: number | false } = {}
+) {
+  if (typeof error === 'string') {
+    error = new Error(error);
+  }
+
+  if (options.exit === false) {
+    console.error(
+      error instanceof CheckupError ? error.render() : `\n${red('Error')} ${error.message}`
```

Comment:
the else case of this ternary seems vaguely odd to me

I would expect something like:

```suggestion
      error instanceof CheckupError ? error.render() : error
```

(console.error is going to "do the right thing" with the actual error itself, we shouldn't pass it a string like that)

Reply:
This is printing custom error messages. This is loosely related to how oclif prints its errors, and this function was originally mimicking what they do in terms of output for general errors, while adding an extra output for checkup specific errors.

Reply:
I'm going to leave this for now, and update at the end of the refactor once I get everything working. I'll likely just remove the custom checkup error.